test.cpp

```
@@ -0,0 +1,132 @@
+    }
+}
+
+float calculateAverage(Student* students, int count) {
+    float sum;
+    for (int i = 0; i < count; i++) {
+        sum += students[i].grade;
+    }
+    return sum / count;
+}
+
+void searchStudent(Student* students, int count, string name) {
```

Cppcheck (2.10-2)

[performance] Function parameter 'name' should be passed by const reference.

(passedByValue)